src/js/components/Video/__tests__/__snapshots__/Video-test.js.snap

```diff
@@ -158,7 +158,7 @@ exports[`Video autoPlay renders 1`] = `
 
 .c5:hover {
   background-color: rgba(221,221,221,0.4);
-  color: #000000;
+  color: #FFFFFF;
```

Comment:
I'm guessing this is the desired behavior (switching the color from black to white), but wondering what would be the impact on the users.

Reply:
I verified interacting with Video that it's behaving correctly. And I couldn't find any other situations that seemed like a change wasn't correct.